### ISSUE TITLE
Hints should hide word suffixes

### ISSUE THREAD

**Issue by joequery** [joequery/quizlet#2]
`hintify("multiple [horse]s")` currently returns `"multiple [h____]s"`. To avoid revealing too much, it should return `"multiple [h____]"`

**Comment by fabianmendes** [comment]
Right, it's properly better! Good. Why not to Fork it? Excuse me, hi haha! I see that it is from 2,013. haha what about to improvements.

Pleased to meet you,
V. Fabian T. Mendes C.
